js/theme/src/global.css

```diff
@@ -1,3 +1,22 @@
+/*
+ Custom element styles
+ */
+gradio-app {
```

Comment:
If this is OK, this hardcoded `style` of the normal Gradio can be removed?
https://github.com/gradio-app/gradio/blob/465f58957f70c7cf3e894beef8a117b28339e3c1/js/app/index.html#L72